src/core/org/apache/jmeter/gui/SimpleCompoundEdit.java

```diff
@@ -0,0 +1,14 @@
+package org.apache.jmeter.gui;
+
+import javax.swing.undo.CompoundEdit;
+
+public class SimpleCompoundEdit extends CompoundEdit {
+
+    public boolean isEmpty() {
+        return size() == 0;
```

Comment:
To `edits`? Seems a matter of taste. I would prefer to limit access to the protected variable.